### ISSUE TITLE
Ajax/DB: Support for search limit 

### ISSUE THREAD

**Issue by heatherleaf** [MUSTE-Project/MULLE#45]
The option(s) for limiting search depth/size (currently `--limit-X`), should be possible to call from the GUI too. Or rather, the limits should be stored in the lessons database (different grammars could have different optimal limits).

**Comment by fredefox** [comment]
In the same vein I was thinking if it would be handy to also be able to specify options on the command line á là the CLI. Currently options on the CLI are parsed from the command line using the package `optparse-applicative`, options for the backend are parsed from a configuration file using the package `yaml`. My idea is that any options specified on the command line should take precedence over the config file. This is maybe very much in the nice-to-have-but-dont-need-to-have bucket.

**Comment by fredefox** [comment]
Ah, but your proposal was to store it in the database. I think this is a better idea since it is lesson specific. Should we do that?

**Comment by heatherleaf** [comment]
I think that would be a good idea. Pruning can make it 2-3x faster, but I'm pretty sure that the limit depends a lot on the grammar (= lesson).

Without limit; 2.1s:
```
$ time -p muste cli -G novo_modo/Exemplum -L ExemplumSwe "flickan slår på en dator" | wc -l
     124
real 2.32
user 2.11
```

With limit=4; 1.2s (2x faster):
```
$ time -p muste cli -G novo_modo/Exemplum -L ExemplumSwe --limit-prune 4 --limit-adjunctions 4 "flickan slår på en dator" | wc -l
     124
real 1.21
user 1.09
```

With limit=3; 0.8s (2.5x faster):
```
$ time -p muste cli -G novo_modo/Exemplum -L ExemplumSwe --limit-prune 3 --limit-adjunctions 3 "flickan slår på en dator" | wc -l
     124
real 0.92
user 0.84
```

With limit=2; 0.7s (3x faster) but now we start to lose menu items:
```
$ time -p muste cli -G novo_modo/Exemplum -L ExemplumSwe --limit-prune 2 --limit-adjunctions 2 "flickan slår på en dator" | wc -l
     104
real 0.81
user 0.72
```

**Comment by fredefox** [comment]
I'll start work on this now.

**Comment by fredefox** [comment]
Closed in 7a39dc66007e83af762422ebb61f9b7da684e63c
